retriever/lib/datasets.py

```diff
@@ -28,3 +35,12 @@ def datasets(arg_script=None):
     all_scripts = sorted(all_scripts, key=lambda s: s.lower())
 
     return all_scripts
+
+def license(dataset):
```

Comment:
Agreed. It could also be used as a check in `install`, `download`, etc. Are you envisioning a structure like:

```
if dataset_exists(dataset):
    do stuff
```

or we could go with:

```
check_dataset_exists()
do stuff
```

And have the function throw the error?